### PR TITLE
Add std CI v2

### PR DESCRIPTION
**Pull request by gbenhaim**
automation: Customize deployment
- Allow to specify cluster version
- Allow to specify ansible moduels version

automation: Adding stdci V2
- stdci V2 will allow us to run several check-patch
  scripts in parallel.
- For now, run the tests on both Openshift origin 3.7 and 3.9.

### CODE REVIEW

File: automation/check-patch.openshift_3-9.packages [hunk 1]
```diff
@@ -0,0 +1 @@
+check-patch.packages
```
[the remark this answers is not in the document]
Comment (gbenhaim) [reply automation/check-patch.openshift_3-9.packages]: Let's adjust the playbooks to support 2.4 in the next week.

**Comment by gbenhaim** [comment]
Ignore the CI results of `check-patch.openshift_3.9`, it runs 3.7 by mistake.

**Comment by cynepco3hahue** [comment]
@gbenhaim did you disable check for docker storage?
 openshift_disable_check=...,docker_storage,...

**Comment by gbenhaim** [comment]
@cynepco3hahue this is the issue: https://github.com/openshift/openshift-ansible-contrib/issues/865

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
@lukas-bednar sure

**Comment by gbenhaim** [comment]
@lukas-bednar In which section of the readme should I add the testing matrix? should I add a new section?

**Comment by lukas-bednar** [comment]
@gbenhaim I would create table at the place of `**Tested on CentOS Linux release 7.4 (Core), OpenShift 3.7 and Ansible 2.4.2**` sentence.
Something like:

| Cluster | Operating System | Ansible |
| :----: | :----: | :----: |
| Openshift 3.7 | CentOS 7.4 | Ansible 2.4.2 |
| Openshift 3.9 | CentOS 7.4 | Ansible 2.4.2 |

Or something like that.

**Comment by gbenhaim** [comment]
part of https://github.com/kubevirt/kubevirt-ansible/issues/44

**Pull request merged**